packages/@jsii/dotnet-runtime/src/Amazon.JSII.Analyzers/JsiiOptionalAnalyzer.cs

```diff
@@ -15,8 +16,8 @@ public class JsiiOptionalAnalyzer : DiagnosticAnalyzer
         private const string Title = "A required property is missing or null";
         private const string MessageFormat = "The property is required and cannot be null";
         private const string MessageFormatWithPropertyName = "The property {0} is required and cannot be null";
```

Comment:
You added periods to the two below messages; any reason not to add them to these?

Reply:
Basically it's the following rules from the Roslyn analyzer for Roslyn analyzers (so meta):
- [RS1033](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md#rs1033-define-diagnostic-description-correctly)
  > The diagnostic description should be one or multiple sentences ending with a punctuation sign
- [RS1032](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md#rs1032-define-diagnostic-message-correctly)
  > The diagnostic message should not contain any line return character and should either be a single sentence without a trailing period or a multi-sentences with a trailing period